dev-java/jackcess/jackcess-1.2.14.3.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Pure Java library for reading from and writing to MS Access databases"
+HOMEPAGE="http://jackcess.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}-sources.jar"
+
+LICENSE="Apache-2.0"
+SLOT="1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+
+CDEPEND=">=dev-java/commons-lang-2.6:2.1
+	>=dev-java/commons-logging-1.1.3:0
+	>=dev-java/log4j-1.2.7:0"
+
+DEPEND=">=virtual/jdk-1.6
+	app-arch/unzip
+	${CDEPEND}"
+
+RDEPEND=">=virtual/jre-1.6
+	${CDEPEND}"
+
+JAVA_GENTOO_CLASSPATH="
+	commons-lang-2.1
+	commons-logging
+	log4j
+"
+
+src_compile() {
+	java-pkg-simple_src_compile
+	java-pkg_addres "${PN}.jar" . \( \
```

Comment:
The extra find arguments are not needed because the only other files contained in the source jar are automatically ignored by java-pkg_addres. It won't hurt if an extra file slips in later but it will hurt if you miss one.